app/_src/production/dp-config/transparent-proxying.md

```diff
@@ -60,7 +60,7 @@ The host that will run the `kuma-dp` process in transparent proxying mode needs
    useradd -u 5678 -U kuma-dp
    ```
 
-3. Redirect all the relevant inbound, outbound and DNS traffic to the {{site.mesh_product_name}} data plane proxy.
+3. Redirect all the relevant inbound, outbound and DNS traffic to the {{site.mesh_product_name}} data plane proxy (if you're running any other services on that machine you need to adjust the comma separated lists of `--exclude-inbound-ports` and `--exclude-outbound-ports` accordingly).
```

Comment:
Isn't the right config something with uid too?

Reply:
would you like to exclude it from redirects completely? like with `--kuma-dp-uid`? we'd need another switch for that like `--redirect-disabled-uids`?

Reply:
@lahabana WDYT?

Reply:
let's discuss it in an issue https://github.com/kumahq/kuma/issues/10672